package.json

```diff
@@ -73,7 +74,7 @@
     "solhint": "^3.3.4",
     "solhint-plugin-prettier": "^0.0.5",
     "solidity-coverage": "^0.7.17",
-    "ts-node": "^9.0.0",
+    "ts-node": "^10.8.1",
```

Comment:
did you check if the tests were still working with mocha explorer with this change ?

Reply:
It works, but in any case we can do it in a different PR => removed the change.